src/EditorFeatures/Core/Implementation/IntelliSense/AsyncCompletion/FilterResult.cs

```diff
@@ -12,12 +13,14 @@ internal struct FilterResult : IComparable<FilterResult>
         public readonly CompletionItem CompletionItem;
         public readonly bool MatchedFilterText;
         public readonly string FilterText;
+        public readonly PatternMatch? PatternMatch;
```

Comment:
could you clarify when we do *not* have this value?  specifically, how does it relate to the experiment.  If you're in the experiment will you have this 100% of the time?  And, if you're not, will this always be null?  Or is thie null-ness of this **unrelated** to the experiment?  Thanks!

Reply:
The null-ness is unrelated to the experiment. If you look at `MatchesFilterText` method, you'd notice that we might include items that don't match the filter text in the completion list (e.g. might from MRU), inthose cases, the pattern match would be null.

Reply:
got it.  thanks.  can you doc?

Reply:
Sure will do. Also @ivanbasov suggested to add a `NoMatch` value to the patternmatch kind, thoughts?

https://github.com/dotnet/roslyn/pull/38499#discussion_r321442131